projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java

```diff
@@ -29,40 +27,39 @@
 @Path(CoordConsts.SVC_CFG_WORK_MGR2)
 @Produces(MediaType.APPLICATION_JSON)
 public class WorkMgrServiceV2 {
+
+  private BatfishLogger _logger = Main.getLogger();
+
+  @DefaultValue(CoordConsts.DEFAULT_API_KEY)
+  @HeaderParam(CoordConsts.SVC_KEY_API_KEY)
+  private String _apiKey;
+
   /** Information on the URI of a request, injected by the server framework at runtime. */
   @Context private UriInfo _uriInfo;
 
   /** Returns the list of {@link Container containers} that the given API key may access. */
   @GET
-  @Path("/containers")
```

Comment:
It looks to me like this will lose the `/`. Is  is not important?

Find the documentation, get back to me :)